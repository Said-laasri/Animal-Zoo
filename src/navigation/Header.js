import './Header.scss';

const Header = () => (
  <div className="header">
    <h1>Animal ZOO</h1>
  </div>
);

export default Header;
