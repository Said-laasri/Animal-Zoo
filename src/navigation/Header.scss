.header {
  display: flex;
  flex-direction: column;
  padding-bottom: 3rem;
  background-image: url('https://rare-gallery.com/thumbs/349036-4k-wallpaper.jpg');
  background-repeat: no-repeat;
  background-size: cover;
  text-align: center;
}

.header > h1 {
  font-size: 7rem;
  background: -webkit-linear-gradient(#eee, #333);
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
}

.search {
  font-size: 3rem;
  background: -webkit-linear-gradient(#eee, #333);
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
}

.input {
  width: 20rem;
  font-family: 'Lato', sans-serif;
  height: 2rem;
  border: none;
  font-size: 2rem;
}
