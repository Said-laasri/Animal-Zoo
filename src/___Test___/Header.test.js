import React from 'react';
import Renderer from 'react-test-renderer';
import Header from '../navigation/Header';

// test Header component
it('renders correctly', () => {
  const tree = Renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
