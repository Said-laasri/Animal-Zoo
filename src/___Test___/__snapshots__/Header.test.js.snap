// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="header"
>
  <h1>
    Animal ZOO
  </h1>
</div>
`;
